.github/workflows/testing.yml

```diff
@@ -0,0 +1,32 @@
+name: "Pytest"
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

Comment:
Can you modify this similar to the Main repo?
(In future if we have many branches and PRs on them)

Reply:
ok I will do it

Reply:
done